pkgs/by-name/im/imhex/package.nix

```
@@ -107,6 +148,6 @@ stdenv.mkDerivation rec {
       kashw2
       cafkafk
     ];
-    platforms = platforms.linux;
+    platforms = platforms.unix;
```

Since other platforms are ruled out by your throw [here](https://github.com/NixOS/nixpkgs/pull/330303/files#diff-cb182ce59baa3510525615b5be3ccdca218b8a32b330b315bfd98a29374c04bdR139),

```suggestion
    platforms = platforms.linux ++ platforms.darwin;
```

is more accurate